deploy-hs

```diff
@@ -86,5 +86,9 @@ docker exec $CID python manage.py makemigrations
 echo "*** manage.py migrate ***"
 docker exec $CID python manage.py migrate
 
+docker exec $CID python manage.py migrate hs_core zero
+
+docker exec $CID python manage.py migrate
+
```

Comment:
This will tear down all hs_core tables, destroying data on machines of anyone who runs `deploy-hs`